README.md

```diff
@@ -40,3 +40,10 @@ Happy hacking!
 ## Deploying a new version
 
 Deploying the Node packages to NPM happens automatically on merge to `main` through GitHub Actions. The package is published to `@techdocs/cli` in NPM. Just bump the version number in the package.json file and create a pull request. It will deploy when merged.
+
+
+## Develop
+
+The `techdocs-cli` package currently has a bit of a weird setup. It consists of two monorepos. The first one is the top level monorepo, where each package is listed in the `packages` directory. The second monorepo is a backstage app monorepo which can be found in `packages/embedded-techdocs`.
```

Comment:
> The `techdocs-cli` package currently has a bit of a weird setup

Appreciate the honesty here. 😁

Reply:
Hah, spent way too much time trying to remove that but i gave up. I don't know how the build setup in backstage work well enough to untangle it enough to put into another monorepo. :)